client/reader/discover/style.scss

```diff
@@ -46,21 +47,24 @@
 		margin-right: 6px;
 	}
 
-	.gridicons-globe {
-		vertical-align: text-bottom;
-		background-color: lighten( $gray, 20% );
-		color: $white;
+	.site-icon {
 		margin-right: 6px;
+		background: lighten( $gray, 20% );
+		.gridicon {
```

Comment:
Nitpick: I would add a space before `.gridicon`. Looks good otherwise.

Reply:
Thanks @jancavan. I've added a new line, and also removed some old gridicon arrow styles we no longer use.